sci-calculators/bc-gh/bc-gh-1.2.6.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://github.com/gavinhoward/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE=""
+
+S="${WORKDIR}/bc-${PV}"
+
+DEPEND=""
+RDEPEND="${DEPEND}"
```

Comment:
This seems rather useless. You can leave these variables empty, or remove them altogether if nothing goes into them.

Reply:
Removed.